### ISSUE TITLE
Button is Unclickable in Windows Touch Screen Device

### ISSUE THREAD

**Issue by hockyy** [rcaferati/react-awesome-button#147]
I'm using Windows 11 in Surface Laptop Studio. The button is unclickable, I can only press it with my touch screen.

BTW, this bug made all app/website that uses this broken/unusable with mouse and/or touchpads

**Comment by hockyy** [comment]
Up

**Comment by hockyy** [comment]
this wasnt an issue in version 6

**Comment by teo-anastasiadis** [comment]
Yep it's an issue with latest version. Will the library accept new merge requests? I can make a hot fix cause I need it as well.

**Comment by hockyy** [comment]
> Yep it's an issue with latest version. Will the library accept new merge requests? I can make a hot fix cause I need it as well. @rcaferati

he aint maintaining the repo anymore :/ might as well just fork it

**Comment by hockyy** [comment]
> Yep it's an issue with latest version. Will the library accept new merge requests? I can make a hot fix cause I need it as well. @rcaferati

can you push
